src/client/components/app/LabDetail/LabDetailPanels.tsx

```diff
@@ -0,0 +1,287 @@
+  right: 0;
+  top: 0;
+  width: 50%;
+`;
+
+const OverviewInfo = styled(Card)`
+  padding: ${({ theme }) => theme.card.padding};
+  flex-shrink: 0;
+
+  a {
+    text-decoration: underline;
+    color: inherit;
+    color: ${({ theme }) => theme.colors.onSurfaceSH1};
+  }
+`;
+
+const StyledText = styled(Text)`
+  display: block;
+  color: ${(props) => props.theme.colors.secondary};
+  white-space: initial;
+`;
+
+const StyledLink = styled.a`
+  display: block;
+  text-decoration: underline;
+  color: inherit;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+`;
+
+const InfoValueRow = styled.div`
+  display: grid;
+  grid-template-columns: 9.6rem 1fr;
+  grid-gap: 0.6rem;
+  white-space: nowrap;
+  color: ${({ theme }) => theme.colors.onSurfaceSH1};
+  font-size: 1.4rem;
+  align-items: center;
+
+  > * {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+`;
+
+const TextWithIcon = styled.div`
+  display: flex;
+  align-items: center;
+
+  ${StyledText} {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    width: 10rem;
+  }
+`;
+
+const StyledIcon = styled(Icon)`
+  margin-left: 1rem;
+  flex-shrink: 0;
+`;
+
+const InfoValueTitle = styled(Text)`
+  font-size: 1.8rem;
+  font-weight: bold;
+  margin-bottom: 0.3rem;
+  color: ${(props) => props.theme.colors.secondary};
+`;
+
+const TokenInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+`;
+
+const TokenLogo = styled(Card)`
+  padding: 2.2rem;
+  height: min-content;
+`;
+
+const OverviewTokenInfo = styled.div`
+  display: grid;
+  grid-template-columns: min-content 1fr;
+  grid-gap: 4.7rem;
+`;
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: ${({ theme }) => theme.layoutPadding};
+`;
+
+const LabOverview = styled(Card)`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  align-self: stretch;
+  max-width: 100%;
+
+  > div:not(:first-child) {
+    margin-top: ${({ theme }) => theme.card.padding};
+  }
+
+  @media ${device.mobile} {
+    ${OverviewTokenInfo} {
+      grid-gap: 2rem;
+    }
+    ${InfoValueRow} {
+      display: flex;
+      margin-top: 0.5rem;
+      flex-direction: column;
+    }
+  }
+
+  @media (max-width: 360px) {
+    ${OverviewTokenInfo} {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+`;
+
+export interface LabDetailPanelsProps {
+  selectedLab: GeneralLabView;
+  displayAddToken?: boolean;
+  currentNetwork?: Network;
+  blockExplorerUrl?: string;
+}
+
+const getTooltip = ({ apyMetadata, address }: Pick<GeneralLabView, 'apyMetadata' | 'address'>) => {
+  if (!apyMetadata) {
+    return null;
+  }
+
+  return (
+    <Tooltip placement="bottom" tooltipComponent={<ApyTooltipData apy={apyMetadata} address={address} />}>
+      <StyledIcon Component={InfoIcon} size="1.5rem" />
+    </Tooltip>
+  );
+};
+
+export const LabDetailPanels = ({
```

Comment:
This and the `LabDetail` share a lot in common with their Vault counterpart, do you think it's worth to DRY so they share the same code? I wasn't sure if the cleaning up was worth it if we are not planning in adding more of these master/detail screens